circle.yml

```diff
@@ -595,10 +591,10 @@ jobs:
             rustup toolchain install nightly-x86_64-unknown-linux-gnu
       - run:
           name: Build (debug mode)
-          command: RUSTFLAGS="-Z sanitizer=address" ASAN_OPTIONS=detect_leaks=1 cargo +nightly build --target x86_64-unknown-linux-gnu
```

Comment:
Ok since this on by default on Linux (not on MacOS): https://clang.llvm.org/docs/AddressSanitizer.html#memory-leak-detection